packages/discovery-styles/package.json

```diff
@@ -7,6 +7,7 @@
   "repository": "https://github.com/watson-developer-cloud/discovery-components",
   "main": "scss/index.scss",
   "scripts": {
+    "prebuild": "scripts/update-styles.sh",
```

Comment:
Does `prebuild` script actually get called? Things changed with yarn v2 and some of these no longer get run automatically. See this [migration item](https://yarnpkg.com/getting-started/migration#explicitly-call-the-pre-and-post-scripts). I also don't see `prebuild` mentioned in the supported [lifecycle scripts](https://yarnpkg.com/advanced/lifecycle-scripts).

Reply:
I didn't intend to run this script automatically. This script need to be run only when the `pdfjs-dist` package is upgraded.